test/testdrive/mzcompose.py

```diff
@@ -83,8 +83,10 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         forward_buildkite_shard=True,
         kafka_default_partitions=args.kafka_default_partitions,
         aws_region=args.aws_region,
-        validate_postgres_stash="materialized",
+        postgres_stash="materialized",
+        validate_catalog_store="shadow",
         default_timeout=args.default_timeout,
+        volumes_extra=["mzdata:/mzdata"],
```

Comment:
What if you forget the volumes_extra? Will there be a cryptic error message?

Reply:
No, Testdrive will just hang on the first DDL and timeout after 30 minutes with no errors. If you run it locally with warnings turned on, you'll see the following warning repeatedly printed:

```
logic error: fetch_current_state refetch expects earliest live diff to advance: v1 vs v1
```

@danhhz might be able to provide more detail, but Testdrive is looking for some files inside of it's container that were written by the Materialized service and not finding any. Allowing them to share a volume fixes the problem.

Reply:
Yeah, the issue is that without this, persist in the two processes would be talking to the same crdb ("consensus"), but different filesystems ("blob"). Another fix would be the external_minio option, but I assume that comes with a perf hit that isn't really necessary